qa/common/gen_qa_ragged_models.py

```diff
@@ -754,7 +750,6 @@ def create_batch_input_models(models_dir):
         import tensorrt as trt
     if FLAGS.graphdef or FLAGS.savedmodel:
         import tensorflow as tf
-        from tensorflow.python.framework import graph_util
```

Comment:
Because it wasn't used. I removed a bunch of unused imports (likely copied from another file, used at some point but the related code was later removed, etc.).